Sources/MongoDriver.swift

```diff
@@ -2,103 +2,99 @@ import Fluent
 import MongoKitten
 
 public class MongoDriver: Fluent.Driver {
+    
     /**
-        MongoDB uses `_id` as the main identifier.
-    */
-    public var idKey: String = "_id"
-
-    /**
-        Describes the types of errors
-        this driver can throw.
-    */
-    public enum Error: ErrorProtocol {
+     Describes the types of errors
+     this driver can throw.
+     */
+    public enum Error: Swift.Error {
         case unsupported(String)
     }
     
-    var database: MongoKitten.Database
-
+    let database: MongoKitten.Database
+    
     /**
-        Creates a new `MongoDriver` with
-        the given database name, credentials, and port. 
-    */
+     Creates a new `MongoDriver` with
```

Comment:
Comment formatting needs to be fixed up a bit.

Should look like this:

```
/**
    Comments are tabbed in 
    4 spaces
*/
```

Not the closing comment does not have a space before it.